site/docs/v1/tech/core-concepts/entity-management.adoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: doc
+title: Entity Management
+description: An overview of FusionAuth Entity Management
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+There are many use cases where it is helpful to model entities in addition to users. Examples might include devices, cars, computers, customers, companies, etc. +
+
+[%hardbreaks]
+
+_Enter Entities._ Entities allow you to model everything right from FusionAuth!
+
+* <<Features>>
+* <<Common Applications>>
+* <<Entity Types Homepage>>
+* <<Entity Homepage>>
+
+=== Features
+
+==== Scalability
+- FusionAuth Entity Management supports large volumes of Entities right out of the box.
+- This is especially helpful for Internet of Things (IOT) devices; FusionAuth scales right alongside them.
+
+==== Typecasting
+- Entities can have types.
+- For example, `locks`, `cars`, `companies`, `corporate divisions`, `computers`, `APIs`
+- Entity Types can help define permissions.
```

Comment:
`Entity Types can help define permissions` => `Entity Types can define permissions`

Reply:
Just caught my eye, but I don't know a ton about Entities.  An Entity type is plural?  or singular?  Feels more natural that `- Entities can have a type`.  In which case an Entity would be of type `lock`, `car`, etc.  I could be wrong, in which case just ignore me :).